packages/wdio-selenium-standalone-service/src/launcher.js

```diff
@@ -36,11 +36,17 @@ export default class SeleniumStandaloneLauncher {
          * update capability connection options to connect
          * to standalone server
          */
-        (
-            Array.isArray(this.capabilities)
-                ? this.capabilities
-                : Object.values(this.capabilities)
-        ).forEach((cap) => Object.assign(cap, DEFAULT_CONNECTION, { ...cap }))
+        if (config.parallelMultiremote) {
```

Comment:
This change should be made in other services too where we modify the capabilities.